app/views/merchant/dashboard/index.html.erb

```diff
@@ -5,7 +5,32 @@
   <h3><%= @merchant.address %></h3>
   <h3><%= "#{@merchant.city}, #{@merchant.state} #{@merchant.zip}" %></h3>
 </center>
+<br><br>
+
+<h1 align = "center">Pending Orders</h1>
+<center>
+  <table>
+    <tr>
+      <th>Order ID</th>
+      <th>Order Date</th>
+      <th>Quantity of Items</th>
+      <th>Order Total</th>
+    </tr>
+    <% unless @merchant.no_orders? %>
```

Comment:
Is this instance method defined in merchant model?

Reply:
It's defined in the merchant/dashboard_controller. Is that okay? I have @merchant_user, @merchant, and @pending_orders defined there.